slution/account_invoice_form/report/account_invoice_form.xml

```diff
@@ -0,0 +1,235 @@
+<odoo>
+    <!-- Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)-->
+    <template id="slution_invoice_form">
+        <div class="header">
+            <div class="row">
+                <div class="col-8">
+                    <h2><span t-field="o.company_id.name"/></h2>
+                </div>
+                <div class="col-4 text-center">
+                    <h3>ใบยืมของชั่วคราว</h3>
+                    <h5>ORIGINAL</h5>
+                </div>
+            </div>
+            <div class="row mt16">
+                <div class="col-8">
+                    <div class="row">
+                        <div class="col-3">
+                            <p>ลูกค้า</p>
+                        </div>
+                        <div class="col-9">
```

Comment:
Can we use address tag instead of this ?

![Selection_273](https://user-images.githubusercontent.com/24691983/79067453-965b2000-7ce9-11ea-9fcd-2413a52b350d.png)

Reply:
If we use address tag, address will show `zip` before `country`.